app/models/ability.rb

```diff
@@ -0,0 +1,15 @@
+class Ability
+  include CanCan::Ability
+
```

Comment:
- You have done great so far :+1: :+1: But there are some requirements you need to meet before moving to the next milestone. :+1: :100: 
`Install CanCanCan in your project.`

- [ ] According to the project requirement, you were meant to install `cancancan` in order to be able to use the authorization rules. But looking at your `Gem File`, the gem is not present. I'll suggest you install the appropriate gem and make sure your project works as expected. :+1: :100: